build.gradle

```diff
@@ -27,7 +27,6 @@ buildscript {
     classpath 'com.android.tools.build:gradle:2.2.3'
     classpath 'com.github.dcendents:android-maven-gradle-plugin:1.4.1'
     classpath 'com.google.gms:google-services:3.0.0'
-    classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
```

Comment:
I was going to replace this, as we can now just use `annotationProcessor` but actually it wasn't being used at all.